### PR TITLE
Add Psc07 test

### PR DESCRIPTION
**Pull request by tlawrencech**
Adds PSC07 test, error validation & submits form to check confirmation of submission page

### CODE REVIEW

File: cypress/support/page_objects/CeasePsc07Page.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        const dayElement = '#register-date-container > div.dateselector > .date-selector > :nth-child(2) > .selector-day';
+        const monthElement = '#register-date-container > div.dateselector > .date-selector > :nth-child(3) > .selector-month';
+        const yearElement = '#register-date-container > div.dateselector > .date-selector > :nth-child(4) > .selector-year';
+
+        cy.selectTodaysDate(dayElement, monthElement, yearElement);
+    }
+
+    confirmPscCessation() {
+        cy.get('.checkbox > label').should('contain.text',
+            'I confirm that Pkdfv Llp has ceased to be a person of significant control');
```
Comment (rsuller) [inline cypress/support/page_objects/CeasePsc07Page.js]: Just a thought but the name `Pkdfv Llp` is used in a number of places. If for whatever reason that needs to change we may need to limit the changes. We could do something similar to what we've done with the company data in the `fixtures` folder. Thoughts?

Comment (tlawrencech) [reply cypress/support/page_objects/CeasePsc07Page.js]: Yeh I agree that would make it easier to maintain.  I've tried adding a psc.json fixture but can't get it to work so far!

Comment (rsuller) [reply cypress/support/page_objects/CeasePsc07Page.js]: Once you've added the file to the fixtures folder, something like this:
```
{
    "psc_company_name": "PKDFV LLP"
}
```

Then you should be able to do this from a spec file:
```
import { psc_company_name } from '../../fixtures/psc.json';
```

And just call `psc_company_name` where you need to.

Comment (tlawrencech) [reply cypress/support/page_objects/CeasePsc07Page.js]: Thanks - I was missing the braces {} in the import!

Comment (tlawrencech) [reply cypress/support/page_objects/CeasePsc07Page.js]: I've added the psc.json fixture and updated usages of PKDFV LLP to call rle_psc_name.  Also deleted the example.json fixture

**Pull request merged**